### PR TITLE
Add i10n FR support

### LINKED ISSUE #41: Support FR
As a french user, I would like to have the application available in French 🇫🇷 🥖

## What needs to be done

- [ ] Create `l10n/fr.json`
- [ ] Translate everything into French

### PR DESCRIPTION
**Pull request by Adaendra**
## 📖 Description
Add i10n FR support of the application

### ⁉️ Related Issues
closes #41 

### 🖼️ Screenshots:
image

### 🧪 How to test the change?
Starting the application with the current language set to FR on your phone (if you are testing the mobile version) or on your computer (if you are testing the web version).

### ☑️ Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [ ] I have added/updated tests.
- [x] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`.

### CODE REVIEW

**Comment by apomalyn** [comment]
Des que je fini ma PR, je passe sur ta PR pour ajouter iOS 😄

**Comment by apomalyn** [comment]
closes #41 

@Adaendra don't forget to add the `closes` in front of the issue related 😄

File: README.md [hunk 1]
```diff
@@ -37,4 +37,11 @@ Now you are ready to run the application. To do so execute the following command
 
 ```shell
 flutter run
+```
+
+#### Troubleshoot
+##### `dart run build_runner build` -> "version solving failed"
+In case of this issue, use the following command instead : 
+```bash
+flutter pub run build_runner build --delete-conflicting-outputs
```
Comment (apomalyn) [inline README.md:46]:
```suggestion
dart run build_runner build --delete-conflicting-outputs
```

**Review by apomalyn — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (Adaendra) [reply README.md:46]: image
Voilà pourquoi j'ai mis `flutter pub run` et non `dart run` (c'est le problème bizarre que j'ai eu au début)

**Review by apomalyn — APPROVED** [review]

**Pull request merged**